### ISSUE TITLE
Add a config or a game rule to disable player being able to steal from backpacks

### ISSUE THREAD

**Issue by macuguita** [BeansGalaxy/Beans-Backpacks-2#112]
People can steal stuff from your backpack and it is specially frustrating in servers because there’s no way to check who stole someone’s items from a backpack, so I think it would be a good idea to add a config or a game rule to disable other players opening your backpack and stealing from it

**Comment by BeansGalaxy** [comment]
Good idea! I'll likely add the options to stop players from opening your backpack when your offline and opening your backpack all together. 

I don't think I'll disable them from taking items off your back since I like the idea of pick-pocketing + the menu will close if you turn around to face them or if you move about a block away from where you were when they opened it.

**Comment by BeansGalaxy** [comment]
You can now enable locking backpacks if you're not the owner by using `/beansmod gamerule lockBackpackNotOwner` or use `lockBackpackOffline` if you just want to protect your backpack while offline.
